modules/xfem/doc/content/syntax/XFEM/index.md

```diff
@@ -1 +1,78 @@
-!template load file=stubs/moose_system.md.template name=XFEM syntax=/XFEM
\ No newline at end of file
+# XFEM Action System
+
+## Overview
+
+The XFEM Action is an object required when running MOOSE simulations using XFEM. It provides
+general XFEM solution parameters to the `XFEMApp` class. It has several functions:
+
```

Comment:
- Causes an algorithm to be run at defined intervals to split the mesh based on a set of evolving defined interfaces.
- Causes a modified quadrature rule to be used to correctly integrate elements split by these interfaces.
- Creates objects to generate output related to the XFEM algorithm.
- Provides a syntax block where various parameters controlling the behavior of XFEM can be specified by the user, including: (then list the stuff you have here in a nested list)